subsystems/sidecar/lib/model.js

```diff
@@ -21,7 +22,8 @@ module.exports = class Model {
   }
 
   async getBundle (link) {
-    link = hypercoreid.isValid(link) ? hypercoreid.normalize(link) : link // if not valid, it is a path
+    const { protocol, drive } = pearLink(link)
+    link = (protocol === 'pear:' && drive.key) ? hypercoreid.normalize(drive.key) : link
```

Comment:
i dont follow this, the link should be the link, you should not strip the pear part

Reply:
@mafintosh good eye!